radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_LF.js

```diff
@@ -1,45 +1,46 @@
 
 var bluetooth = require('./CommBluetooth');
-//create empty object to be exported
+// create empty object to be exported
 var ReaderArduino_RC522_LF = {};
 
-ReaderArduino_RC522_LF.connectRC522Parsed = function(success, failure, address) {
+ReaderArduino_RC522_LF.connectRC522Parsed = function(success, failure, reader) {
    var tagId, result;
    var returnVal = {};
    var buffer = "";
 
-   //parser function, expects frames "starting with HEADER"
+   // parser function, expects frames "starting with HEADER"
    var parsedResults = function(data){
       buffer = buffer.concat(data);
       var nextFrameStart = buffer.lastIndexOf("~~~~~~~~HEADER~~~~~~~~");
       if(nextFrameStart > 0){
          var currentFrame = buffer.substring(0, nextFrameStart);
          var idIndex = currentFrame.indexOf("Card UID: ");
-         if(idIndex >= 0){//skip past header
+         if(idIndex >= 0){ // skip past header
             idIndex += 10;
             returnVal.id = currentFrame.substring(idIndex, idIndex + 11);
             returnVal.reader = "RC522 LF";
+            returnVal.friendlyName = reader.friendlyName;
          }
-         if(currentFrame.indexOf("responded with NAK") >= 0){//msg given when seen
+         if(currentFrame.indexOf("responded with NAK") >= 0){ // msg given when seen
             returnVal.report = "seen";
             returnVal.time = resources.getCurrentTime();

```

Comment:
There's a clobber target to Resources so there's no need to require it. However, as a style thing, what do you think @vinchan?